newrelic/newrelic-python-agent
# Obfuscate License Keys in Logs

# Overview

* Add obfuscation to all instances where license keys are printed or logged.

# Testing

* Manual testing must be done to ensure logs don't contain the new relic license keys.
  * Run a test app with a valid license key, with debug and audit logs sent to files.
  * Use `grep -r . MYLICENSEKEY` to find any stray license keys.
  * Repeat the above with `newrelic-admin validate-config newrelic.ini agent.log`
  * Repeat the above with `newrelic-admin license-key newrelic.ini agent.log`
